src/librepgp/packet-show.cpp

```diff
@@ -129,13 +129,13 @@ static pgp_map_t pubkey_alg_map[] = {
   {PGP_PKA_RSA_ENCRYPT_ONLY, "RSA Encrypt-Only"},
   {PGP_PKA_RSA_SIGN_ONLY, "RSA Sign-Only"},
   {PGP_PKA_ELGAMAL, "Elgamal (Encrypt-Only)"},
-  {PGP_PKA_DSA, "DSA"},
-  {PGP_PKA_ECDH, "ECDH"},
-  {PGP_PKA_ECDSA, "ECDSA"},
+  {PGP_PKA_DSA, PGP_ALGNAME_DSA},
+  {PGP_PKA_ECDH, PGP_ALGNAME_ECDH},
+  {PGP_PKA_ECDSA, PGP_ALGNAME_ECDSA},
   {PGP_PKA_ELGAMAL_ENCRYPT_OR_SIGN, "Reserved (formerly Elgamal Encrypt or Sign"},
   {PGP_PKA_RESERVED_DH, "Reserved for Diffie-Hellman (X9.42)"},
-  {PGP_PKA_EDDSA, "EdDSA"},
```

Comment:
Looks good except there is a side effect that `EdDSA` is changed to `EDDSA`. @dewyatt @ni4 is this acceptable?

Reply:
@ronaldtse Somehow missed your comment before writing my own :) I also don't think that we should change it this way for human-readable output, since common spelling is EdDSA, as well as AES-128, SHA-256.